res/css/photon-components.css

```diff
@@ -48,3 +48,48 @@
 .photon-button-ghost:hover:active {
   background-color: var(--grey-90-a20);
 }
+
+/* https://design.firefox.com/photon/components/message-bars.html */
+.photon-message-bar {
+  width: 100%;
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 1.4;
+  border-radius: 4px;
+  min-height: 32px;
+  display: flex;
+  align-items: center;
+  padding: 0 4px;
+  box-sizing: border-box;
+}
+
+.photon-message-bar-error {
+  background-color: var(--red-60);
+  color: #fff;
+}
+
+.photon-message-bar-error > * {
+  margin: 0 4px;
+}
+
+.photon-message-bar-error-icon {
+  display: inline-block;
+  height: 15px;
+  width: 15px;
+  background-image: url(../img/svg/error.svg);
+  flex-shrink: 0;
+}
+
+.photon-message-bar-error-action {
+  background-color: var(--red-70);
+  font-size: 11px;
+  border: none;
+  border-radius: 4px;
+  color: #fff;
+  padding: 6px;
+  margin-left: 8px;
+}
```

Comment:
Except for the colors and the margin, the style should go to a button section and it should be a `.photon-button-micro` according to the spec: 
![image](https://user-images.githubusercontent.com/454175/51552699-71bf2200-1e71-11e9-94ec-a42b929d172f.png)
Micro buttons are defined here: https://design.firefox.com/photon/components/buttons.html

From what I see, here are the differences to what you did here:
```css
padding: 0 8px;
height: 24px;
border-radius: 2px;
```

But actually it's both a `photon-button` and a `photon-button-micro` and so I believe we need to define the height only for photon-button-micro, other things are already there.